examples/autoconfigure/src/main/java/io/opentelemetry/example/autoconfigure/AutoConfigExample.java

```diff
@@ -0,0 +1,59 @@
+package io.opentelemetry.example.autoconfigure;
+
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.api.metrics.GlobalMetricsProvider;
+import io.opentelemetry.api.metrics.LongCounter;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.api.trace.Tracer;
+import io.opentelemetry.sdk.autoconfigure.OpenTelemetrySdkAutoConfiguration;
+
+/**
+ * An example of using {@link io.opentelemetry.sdk.autoconfigure.OpenTelemetrySdkAutoConfiguration}
+ * and logging exporters: {@link io.opentelemetry.exporter.logging.LoggingSpanExporter} and {@link
+ * io.opentelemetry.exporter.logging.LoggingMetricExporter}.
+ */
+public final class AutoConfigExample {
+  private static final String INSTRUMENTATION_NAME = AutoConfigExample.class.getName();
+
+  public static void main(String[] args) throws InterruptedException {
+    // Export metrics every 800ms to avoid long Thread.sleep() at the end
+    // You probably won't need to export metrics so often in a real life scenario
+    System.setProperty("otel.imr.export.interval", "800");
+
+    // Let the SDK configure itself using environment variables and system properties
+    OpenTelemetrySdkAutoConfiguration.initialize();
+
+    // OpenTelemetrySdkAutoConfiguration sets global instances of OpenTelemetry and MetricsProvider
+    // AutoConfigExample uses them instead of passing OpenTelemetry around by hand
+    AutoConfigExample example = new AutoConfigExample();
+    // Do some real work that'll emit telemetry
+    example.doWork();
+
+    // Flush out the metrics that have not yet been exported
+    Thread.sleep(1000);
+  }
+
+  private final Tracer tracer;
+  private final LongCounter counter;
+
+  public AutoConfigExample() {
+    this.tracer = GlobalOpenTelemetry.get().getTracer(INSTRUMENTATION_NAME);
```

Comment:
I think we'd greatly prefer that our examples don't use the global instance, if at all possible. Usage of the global should only be for extreme circumstances, or for use alongside the agent. For manual instrumentation, all of our examples should use instances of OpenTelemetry and MetricsProvider.

Reply:
You will have to get it from the global, but that can be done separately. Since autoconfigure doesn't give you a MeterProvider, should it even be in this example?

Reply:
Hmm, good point. Removed metrics from this example -- we can add them back once autoconfigure (or `OpenTelemetry`) returns the created `MetricsProvider`.